src/commands/scan.yml

```diff
@@ -63,6 +68,19 @@ parameters:
     type: string
     default: "10m"
 steps:
+  - run:
+      name: Store Snyk CLI version as a temporary checksum file
+      environment:
+        SNYK_VERSION: <<parameters.cli-version>>
+      command: |
+        if [[ -z "${SNYK_VERSION}" ]]
+          curl https://static.snyk.io/cli/latest/version > /tmp/.snyk-version
```

Comment:
There is a potential race-condition when the version is received here, and the later download which could already have a newer version. It would be better to fix the version as an output parameter

Reply:
Feedback taken!